docs/organization/organisation.md

```diff
@@ -0,0 +1,142 @@
+# Online Banking Organization Page
+
+## Project Overview
+This project aims to develop a mobile banking application tailored for Cooperative Financial
+Institutions (CFIs). The application will be open-source and designed to integrate seamlessly with
+various core banking systems (CBS), ranging from simple batch-based systems to advanced live
+systems like adorsys Ledgers Mambu and Mifos.
+
+## Project goals
+this online banking project is aimed at building an application with the following goals.
+1. Enhance financial accessibility and inclusivity for CFIs.
+2. Provide a secure, flexible, and scalable mobile banking platform.
+3. Facilitate seamless integration with existing core banking systems.
+4. High security, with features such as cryptographic signing and role-based access control.
+5. Modularity and extensibility to support future development and integration.
+6. User-friendly design that can be easily translated into multiple languages.
+
+##Project Constraints
+1.The application must comply with ISO20022 data standards to simplify connectivity to
+different core banking systems.
+2. Compatibility with both live and non-live core banking systems is essential.
+3. The platform must support a range of devices, from older smartphones to desktops.
```

Comment:
These data is redundant..
project goals and constraints are mentioned already in the project architecture..
you can redirect using a **link**.
or better still, the project description alone is better for the organization page, no need for goals or constrains in here since it has been mentioned in the project architecture already